tests/pytests/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ IF (RS_VERBOSE_TESTS)
 ENDIF()
 
 FILE(GLOB PY_TEST_FILES "test*.py")
-SET(baseCommand "MODARGS+='timeout 100000;' ${CMAKE_CURRENT_SOURCE_DIR}/runtests.sh --module=${RS_TEST_MODULE_SO} ${RS_TEST_ARGS}")
+SET(baseCommand "MODARGS+='timeout 0;' ${CMAKE_CURRENT_SOURCE_DIR}/runtests.sh --module=${RS_TEST_MODULE_SO} ${RS_TEST_ARGS}")
```

Comment:
What is this change means?

Reply:
I have changed the default of `100000` to `0` using the new feature.
We had to put the default because some tests have timed out on the sanitizer tests.